frontend/control-center/public/start.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Airy Start Page</title>
+  <meta charset="utf-8" />
+  <link rel="shortcut icon" href="/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="theme-color" content="#000000" />
+  <link rel="manifest" href="/manifest.json" />
+  <link href="http://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">
```

Comment:
@AitorAlgorta This is where the import needs to be fixed. The original `start.html` was moved here :)

Reply:
You right!